airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -582,6 +582,34 @@ def decorator_f(self, *args, **kwargs):
 
         return retry_decorator
 
+    @staticmethod
+    def get_ui_field_behaviour() -> Dict[str, Any]:
+        """Returns custom UI field behaviour for AWS Connection."""
+        return {
```

Comment:
Very minor nitpick, the flipping back and forth between single and double quotes in the return value is odd.

Reply:
Good point. I'm forget that `black` in airflow configure with `skip-string-normalization = true`